python/ql/lib/semmle/python/Stmts.qll

```diff
@@ -364,10 +387,10 @@ class Try extends Try_ {
     result = this.getAnOrelse()
   }
 
-  override ExceptStmt getHandler(int i) { result = Try_.super.getHandler(i) }
+  override Stmt getHandler(int i) { result = Try_.super.getHandler(i) }
 
   /** Gets an exception handler of this try statement. */
-  override ExceptStmt getAHandler() { result = Try_.super.getAHandler() }
+  override Stmt getAHandler() { result = Try_.super.getAHandler() }
```

Comment:
It's probably not too serious, but strictly speaking both this and the change on line 390 are breaking changes, as they generalise the result type (and so code that calls a method that's available on `ExceptStmt` but not `Stmt` now needs an inline cast.

Reply:
I am trying the common superclass idea that you suggested.